code/__DEFINES/supermatter.dm

```diff
@@ -36,7 +36,7 @@
 #define MATTER_POWER_CONVERSION 10 //Crystal converts 1/this value of stored matter into energy.
 
 //These would be what you would get at point blank, decreases with distance
-#define DETONATION_HALLUCINATION 600
+#define DETONATION_HALLUCINATION 1200 SECONDS
```

Comment:
I don't know in what units the old number was in, but for the new one minutes might be more apt.